deploy/k8s/charts/trustification/templates/services/report/020-Configmap.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and .Values.modules.report.enabled }}
+{{- $mod := dict "root" . "name" "report-server" "component" "report" "module" .Values.modules.report -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "trustification.common.name" $mod }}-nginx-config
+data:
+  nginx.conf: |
+    worker_processes auto;
+    error_log /var/log/nginx/error.log warn;
```

Comment:
This might lead to a "disk-full" situation with no one actually reading the logs. I think the while file should be aligned with the settings from the spog-ui nginx: https://github.com/trustification/trustification/blob/main/spog/ui/config/nginx.conf

Including:

```
error_log /dev/stdout info;
server_name  _;
```

Reply:
ok